bfvmm/src/vcpu/src/vcpu_intel_x64.cpp

```diff
@@ -55,11 +56,20 @@ vcpu_intel_x64::vcpu_intel_x64(int64_t id,
 vcpu_error::type
 vcpu_intel_x64::start()
 {
-    // if (m_vmm.start() != vmm_error::success)
-    //     return vcpu_intel_x64_error::failure;
+    std::cout << "About to start and launch the hypervisor" << std::endl;
```

Comment:
this extra cout should be removed. Text that we plan to keep around forever should also use the debug.h stuff instead of cout manually